config.py

```diff
@@ -23,6 +23,9 @@ class Config:
     # Default viewport is bounding box for Philadelphia, PA
     VIEWPORT = '39.861204,-75.310357|40.138932,-74.928582'
 
+    OPEN_WEATHER_MAP_API_KEY = os.environ.get('OPEN_WEATHER_MAP_API_KEY') or \
+        'SjefBOa$1FgGco0SkfPO392qqH9'
```

Comment:
is this our api key?? probably shouldn't commit this. once we merge in this branch we should let everyone know to add the api key to our local configs